musl-cross.nix

```diff
@@ -0,0 +1,5 @@
+{ src ? ./default.nix }:
```

Comment:
Errr should we have  new top level files? I do want an easier way to get people going, but not sure what should be it.

Reply:
Oh no! This was just for my testing--especially early on before recent improvements -- to avoid error-prone and/or long nix incantations.  This should 0% be merged anywhere, I'll drop it sorry.